v3/examples/contextmenus/main.go

```diff
@@ -28,7 +28,7 @@ func main() {
 	mainWindow := app.NewWebviewWindowWithOptions(application.WebviewWindowOptions{
 		Title:  "Context Menu Demo",
 		Width:  1024,
-		Height: 1124,
+		Height: 1024,
```

Comment:
I set this to 1124 as all the boxes didn't fit on Windows :-) Do they all fit in Mac?

Reply:
![image](https://github.com/wailsapp/wails/assets/1943904/be02482b-9afb-4e40-ad59-87970187e4bf)

Reply:
I just sized it and then ran `wails.Window.Size()` in devtools and got this:
```
height: 1681,
width:1024
```

Reply:
On my screen a little bit was hidden under the windows taskbar, so made it smaller to fit and scrollbars with appear, I don't know how it looks on Mac.

Reply:
Ah the joys of different screen resolutions 😅